frontend/src/pages/NewsfeedPage.tsx

```diff
@@ -1,29 +1,54 @@
 import React, {useCallback, useEffect, useState} from "react";
 import {Susume} from "../service/Model";
-import SusumeComponent from "../components/SusumeComponent";
 import {getSusumesForNewsFeed} from "../service/BlogService";
+import {useNavigate} from "react-router-dom";
+import "../components/NewsfeedSusumeComponent.css"
+import NewsfeedSusumeComponent from "../components/NewsfeedSusumeComponent";
+import "./NewsfeedPage.css"
+import {BottomNavigation, BottomNavigationAction, Paper} from "@mui/material";
 
 export default function NewsfeedPage(){
-
+    const nav = useNavigate()
     const [susumeArray, setSusumeArray] = useState<Array<Susume>>([])
 
     const refreshSusumes = useCallback(() => {
         getSusumesForNewsFeed()
             .then(data => setSusumeArray(data))
-    }, [])
-
-
-    useEffect(() =>{
-        refreshSusumes()
-    }, [refreshSusumes])
-
-    const susumeList = susumeArray.map(susume =><SusumeComponent key={susume.postId} addToSaveList={true} susume={susume} privateList={false}  hasDeleteButton={false} isOnOwnBlog={false} refreshSusumes={refreshSusumes}/>)
-
-
-    return(
-        <div>
-            {susumeList}
-        </div>
-    )
+            .catch((e) => {
+                if(e.response.status === 403){
+                    nav("/")
+                }
+            })
+    }, [nav])
+
+    const logOut = () => {
+        localStorage.removeItem("jwt")
+        nav("/")}
+
+        useEffect(() => {
+            refreshSusumes()
+        }, [refreshSusumes])
+
+        const susumelist2 = susumeArray.map(susume => <NewsfeedSusumeComponent key={susume.postId} susume={susume}
+                                                                               refreshSusume={refreshSusumes}/>)
+
+        return (
+            <>
+                <div className={"newsfeedDiv"}>
```

Comment:
Maybe you could move the 300px margin from NewsFeedSusumeComponent to a padding here.